GVFS/GVFS.Common/Prefetch/Jobs/CheckoutJob.cs

```diff
@@ -31,13 +32,14 @@ public class CheckoutJob : Job
         // Checkout requires synchronization between the delete/directory/add stages, so control the parallelization
         private int maxParallel;
 
-        public CheckoutJob(int maxParallel, IEnumerable<string> folderList, string targetCommitSha, ITracer tracer, Enlistment enlistment)
+        public CheckoutJob(int maxParallel, IEnumerable<string> folderList, string targetCommitSha, ITracer tracer, Enlistment enlistment, bool force = false)

```

Comment:
I think this class should never have been moved to `GVFS.Common`. It is only used by `CheckoutPrefetcher`, so it can be moved back into the `FastFetch` project. This wasn't your change, but would you mind cleaning that up? (Again, the goal being to contain the `force` logic rather than spread it all over`

Reply:
Responding to last 3 comments - all sounds good. Thanks for the quick review!